packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
+# TODO: check logs config with 'docker' or 'docker-json' template
+
+## explorer confs mapping
+
+org.nuxeo.apidoc.apidocAdmin.password=${env:ORG_NUXEO_APIDOC_APIDOCADMIN_PASSWORD}
+
+
+## default confs mapping
+
+nuxeo.data.dir=${env:NUXEO_DATA_DIR:nxserver/data}
+nuxeo.server.dir=${env:NUXEO_SERVER_DIR:nxserver}
+
+# Default IP address binding
+nuxeo.bind.address=${env:NUXEO_BIND_ADDRESS:0.0.0.0}
+
+# publicly visible URL
+nuxeo.url=${env:NUXEO_URL:http://localhost:8080/nuxeo}
+
+# Application context path
+org.nuxeo.ecm.contextPath=${env:ORG_NUXEO_ECM_CONTEXTPATH:/nuxeo}
+
+org.nuxeo.ecm.product.name=Nuxeo Explorer
+
+nuxeo.notification.eMailSubjectPrefix=${env:NUXEO_NOTIFICATION_EMAILSUBJECTPREFIX:[Explorer] }
+
+# Tomcat
+session.timeout=${env:SESSION_TIMEOUT:60}
+session.config.tracking.mode.cookie=${env:SESSION_CONFIG_TRACKING_MODE_COOKIE:true}
+
+
+## common-base confs mapping
+
+nuxeo.server.http.port=${env:NUXEO_SERVER_HTTP_PORT:8080}
+nuxeo.server.http.maxThreads=${env:NUXEO_SERVER_HTTP_MAXTHREADS:20}
+nuxeo.server.http.acceptCount=${env:NUXEO_SERVER_HTTP_ACCEPTCOUNT:1000}
+nuxeo.server.http.connectionUploadTimeout=${env:NUXEO_SERVER_HTTP_CONNECTIONUPLOADTIMEOUT:60000}
+nuxeo.server.ajp.enabled=${env:NUXEO_SERVER_AJP_ENABLED:false}
+nuxeo.server.ajp.port=${env:NUXEO_SERVER_AJP_PORT:8009}
+nuxeo.server.ajp.secretRequired=${env:NUXEO_SERVER_AJP_SECRETREQUIRED:true}
+nuxeo.server.ajp.secret=${env:NUXEO_SERVER_AJP_SECRET:changeme}
+nuxeo.server.https.port=${env:NUXEO_SERVER_HTTPS_PORT:0}
+nuxeo.server.tomcat_admin.host=${env:NUXEO_SERVER_TOMCAT_ADMIN_HOST:localhost}
+nuxeo.server.tomcat_admin.port=${env:NUXEO_SERVER_TOMCAT_ADMIN_PORT:8005}
+nuxeo.server.tomcat_error.show_report=${env:NUXEO_SERVER_TOMCAT_ERROR_SHOW_REPORT:false}
+nuxeo.server.tomcat_error.show_server_info=${env:NUXEO_SERVER_TOMCAT_ERROR_SHOW_SERVER_INFO:false}
+nuxeo.server.jvmRoute=${env:NUXEO_SERVER_JVMROUTE:nuxeo}
+NuxeoContainer.autoactivation=${env:NUXEOCONTAINER_AUTOACTIVATION:false}
+# HSTS configuration is applied when HTTPS is enabled, unless the following property is false
+nuxeo.server.hsts.enabled=${env:NUXEO_SERVER_HSTS_ENABLED:true}
+nuxeo.server.hsts.maxage=${env:NUXEO_SERVER_HSTS_MAXAGE:2592000}
+nuxeo.server.hsts.includesubdomains=${env:NUXEO_SERVER_HSTS_INCLUDESUBDOMAINS:false}
+nuxeo.server.hsts.preload=${env:NUXEO_SERVER_HSTS_PRELOAD:false}
```

Comment:
No reasons to modify these parameters at the docker level for the explorer.

Reply:
removed